specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/networkManagerSecurityConfiguration.json

```diff
@@ -0,0 +1,1349 @@
+        "modelAsString": true
+      }
+    },
+    "SecurityConfigurationRuleDirection": {
+      "type": "string",
+      "description": "The direction of the rule. The direction specifies if rule will be evaluated on incoming or outgoing traffic.",
+      "enum": [
+        "Inbound",
+        "Outbound"
+      ],
+      "x-ms-enum": {
+        "name": "SecurityConfigurationRuleDirection",
+        "modelAsString": true
+      }
+    },
+    "NetworkManagerSecurityGroupItem": {
+      "type": "object",
+      "properties": {
+        "networkGroupId": {
+          "type": "string",
+          "description": "Network manager group Id."
+        }
+      },
+      "description": "Network Manager Security Group Item."
+    },
+    "AddressPrefixItem": {
+      "type": "object",
+      "properties": {
+        "addressPrefix": {
+          "type": "string",
+          "description": "Address Prefix."
+        },
+        "addressPrefixType": {
+          "type": "string",
+          "description": "Address Prefix Type.",
+          "enum": [
+            "IPPrefix",
+            "ServiceTag"
+          ],
+          "x-ms-enum": {
+            "name": "AddressPrefixType",
+            "modelAsString": true
+          }
+        }
+      },
+      "description": "Address Prefix Item."
+    },
+    "NetworkManagerSecurityConfigurationImport": {
+      "properties": {
+        "networkSecurityGroupURI": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "List of nsg uris."
+        },
+        "importDenyRulesAsAdminRules": {
+          "type": "boolean",
+          "description": "Flag if Import Deny Rules As Admin Rules."
+        },
+        "adminSecurityConfigurationUri": {
+          "type": "string",
+          "description": "admin Security Configuration Uri."
+        },
+        "removeAllowVnetInboundRule": {
+          "type": "boolean",
+          "description": "Flag if Remove Allow Vnet Inbound Rule."
+        },
+        "removeAllowAzureLoadBalancerInboundRule": {
+          "type": "boolean",
+          "description": "Flag if Remove Allow Azure Load Balancer Inbound Rule."
+        },
+        "removeAllowVnetOutboundRule": {
+          "type": "boolean",
+          "description": "Flag if Remove Allow Vnet Outbound Rule."
+        },
+        "removeAllowInternetOutboundRule": {
+          "type": "boolean",
+          "description": "Flag if Remove Allow Internet Outbound Rule."
+        }
+      },
+      "description": "Network Manager Security Configuration Import Parameters."
+    },
+    "SecurityConfigurationImportResult": {
+      "properties": {
+        "userSecurityConfiguration": {
+          "type": "string",
+          "description": "User Security Configuration."
+        },
+        "adminSecurityConfiguration": {
+          "type": "string",
+          "description": "Admin Security Configuration."
+        },
+        "failedImports": {
```

Comment:
Does this property represent only a single failedImport object? The property name sounds as multiple will be reported. So cross checking.